src/App.tsx

```diff
@@ -7,27 +7,76 @@ import Oppfolging from './components/oppfolging';
 import PersonaliaBoks from './components/personalia-boks';
 import { Registrering } from './components/registreringsInfo';
 import { Ytelser } from './components/ytelserinfo';
+import { Chips } from '@navikt/ds-react';
+import { useState } from 'react';
 
 export interface AppProps {
     fnr: string;
     enhet?: string;
 }
 
-const App = (props: AppProps) => (
-    <main className="app veilarbdetaljerfs">
-        <StoreProvider fnr={props.fnr}>
-            <PilotAlert />
-            <Nokkelinfo />
-            <div className="main_grid">
-                <CvInnhold />
-                <Jobbonsker />
-                <Oppfolging />
-                <PersonaliaBoks />
-                <Registrering />
-                <Ytelser />
-            </div>
-        </StoreProvider>
-    </main>
-);
+const App = (props: AppProps) => {
+    const options = ['CV', 'Jobbønsker', 'Oppfølging', 'Personalia', 'Registrering', 'Ytelser'];
+
+    const [selectedComponents, setSelectedComponents] = useState<string[]>(options);
+
+    const toggleComponent = (componentName: string) => {
+        setSelectedComponents((prevSelected) =>
+            prevSelected.includes(componentName)
+                ? prevSelected.filter((name) => name !== componentName)
+                : [...prevSelected, componentName]
+        );
+    };
+
+    const renderComponent = (componentName: string) => {
+        switch (componentName) {
+            case 'CV':
+                return <CvInnhold />;
+            case 'Jobbønsker':
+                return <Jobbonsker />;
+            case 'Oppfølging':
+                return <Oppfolging />;
+            case 'Personalia':
+                return <PersonaliaBoks />;
+            case 'Registrering':
+                return <Registrering />;
+            case 'Ytelser':
+                return <Ytelser />;
+            default:
+                return null;
+        }
+    };
+
+    return (
+        <main className="app veilarbdetaljerfs">
+            <StoreProvider fnr={props.fnr}>
+                <PilotAlert />
+                <Nokkelinfo />
+
+                <div className="overblikkChips">
+                    <Chips>
+                        {options.map((c) => (
+                            <Chips.Toggle
+                                key={c}
+                                selected={selectedComponents.includes(c)}
+                                onClick={() => toggleComponent(c)}
+                            >
+                                {c}
+                            </Chips.Toggle>
+                        ))}
+                    </Chips>
+                </div>
+
+                <div className="main_grid">
```

Comment:
Strengt tatt ikke noe som ble innført i denne PR-en men nå har vi brukt tre forskjellige navnekonvensjoner for CSS-klassene våres. Fint om vi kan bestemme oss for _en_, f.eks. `camelCase`. Kan godt tas separat i en egen opprydning PR, men tenkte å bare skrive det.